tdp_vars_defaults/hive/hive_metastore.yml

```
@@ -6,5 +6,8 @@
 hive_ms_credentials_store_path: "{{ hive_ms_conf_dir }}/hive.jceks"
 hive_ms_credentials_store_uri: localjceks://file{{ hive_ms_credentials_store_path }}
 
+compaction_enabled: true
+
 hive_site:
   hadoop.security.credential.provider.path: "{{ hive_ms_credentials_store_uri }}"
+  hive.compactor.initiator.on:  "{{ compaction_enabled and ('hive_compaction_initiator' in group_names ) | string | lower }}"
```

I think we should not use inventory for this use case and use the first Hive MS. This way the user need to change `tdp_vars` to change compactor property and this will trigger stale and reconfigure behavior of `tdp-lib`. `tdp-lib` does not watch topology file to trigger reconfiguration.
```suggestion
  hive.compactor.initiator.on:  "{{ compaction_enabled and (inventory_hostname == groups['hive_ms'][0]) }}"
```